src/main/java/frc/robot/Robot.kt

```diff
@@ -61,6 +64,7 @@ object Robot : LoggedRobot() {
         driveController.rightBumper().whileTrue(Chassis.driveToRightBranch)
         driveController.y().whileTrue(Elevator.manualUp)
         driveController.a().whileTrue(Elevator.manualDown)
+        driveController.start().onTrue(Commands.runOnce({Chassis.resetRotation(Rotation2d()) }))
```

Comment:
[This needs to be from the blue alliance perspective ](https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/swerve/SwerveDrivetrain.html#resetRotation(edu.wpi.first.math.geometry.Rotation2d))